README.md

```diff
@@ -2,30 +2,41 @@
 
 Loads the configuration for your module.
 
+
 ## Usage
 
-Although you can require this module directly, it's recommended you create your own `config.js` file that can be cached by the require system and called *without* a path:
+Although you can require this module directly, it's recommended you create your
+own `config.ts` file that can be cached by the require system and called *without* a path:
 
-```js
-// config.js
-module.exports = require('@snyk/config')('<directory with config files>');
+```typescript
+// config.ts
+import { loadConfig } from 'snyk-config';
+export const config = loadConfig('<directory with config files>');
```

Comment:
guess in `registry` we better still use `export = loadConfig()` instead, otherwise need to touch loads of files

Reply:
As an initial step, sure.